config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -304,3 +304,39 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: pr-apidiff
+  - name: pull-cluster-api-provider-azure-coverage
+    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    always_run: false
```

Comment:
can we make this always run, but not required for the PR to pass?

Reply:
+1

Reply:
done

Reply:
Initially, I added to not run then we can validate before become "official" :)